bin/build-release.sh

```diff
@@ -109,7 +109,11 @@ function package {(
 )}
 
 function dockerImage {(
-  cmd="docker build -t mesos/storm:git-`git rev-parse --short HEAD` ."
+  cmd="docker build \
+       --build-arg MESOS_RELEASE=$MESOS_RELEASE \
+       --build-arg STORM_RELEASE=$STORM_RELEASE \
+       --build-arg MIRROR=$MIRROR \
+        -t mesos/storm:git-`git rev-parse --short HEAD` ."
```

Comment:
@DarinJ  We should also make `mesos/storm` configurable. Something like `<user_name>/<image_name>` where `<image_name>` is something that is in accordance with https://github.com/mesos/storm/issues/83

For now, lets keep that change separate so we don't block this PR on  https://github.com/mesos/storm/issues/83 ?